components/alerts/alerts.css

```diff
@@ -0,0 +1,52 @@
+.alert {
+  font-size: 0.875rem;
+  padding: 1.5rem;
+  margin-bottom: 1rem;
+  border: none;
+  border-radius: 0.5rem;
+  transition: 0.2s ease-in-out;
+  display: flex;
+  color: var(--white);
+  align-items: center;
+  gap: 1rem;
+  box-shadow: var(--shadow-xl);
+}
+
+.alert p {
+  color: var(--white);
+}
+
+.alert .close {
+  margin-left: auto;
+  opacity: 0.7;
+}
+
+.alert .close:hover {
+  opacity: 1;
+  cursor: pointer;
+}
+
+.alert-success {
+  background-color: var(--green-300);
+  color: #fff;
+}
+
+.alert-danger {
+  background-color: var(--red);
+  color: #fff;
+}
+
+.alert-warning {
+  background-color: #ff9800;
+  color: #fff;
```

Comment:
The colors can be save to variables as done above.

Reply:
Ah yes. Will make the change.